walkthroughs/week-2-server/examples/server-stats/src/main/java/com/google/sps/servlets/ServerStatsServlet.java

```diff
@@ -37,7 +37,8 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
 
     // Convert the server stats to JSON
     ServerStats serverStats = new ServerStats(startTime, currentTime, maxMemory, usedMemory);
-    String json = convertToJson(serverStats);
+    // String json = convertToJson(serverStats);
+    String json = convertToJsonUsingGson(serverStats);
```

Comment:
These changes should be in next commit when adding gson dependency into pom.xml. Please keep in mind next time. 

Meanwhile, remove the unused code other than comment the code.